components/eam/src/physics/rrtmgp/radiation.F90

```diff
@@ -103,6 +110,10 @@ module radiation
    ! zeroes out the aerosol optical properties if False
    logical :: do_aerosol_rad = .true.
 
+   ! Logical to indicate if aerosol optical properties are read in directly from
+   ! input file i.e., SPA:
+   logical :: do_SPA_optics = .false.
+
```

Comment:
Just to make sure I'm not missing something: you're initializing to false here but then overwriting with whatever was specified in the namelist, right?

Reply:
That's right.